### PR TITLE
Fix Alt Clicking opening up a window and Add back some object verbs to the browser stat panel

### PR DESCRIPTION
**Pull request by Couls**
## About The Pull Request
Fixies

## Why It's Good For The Game
fixes some more issues with browser stat panel

## Changelog
:cl:
fix: Alt clicking items in the listed turf tab doesn't open up a popup
fix: Toggle suit sensors button missing
/:cl:

### CODE REVIEW

File: html/statbrowser.html [hunk 1]
```diff
@@ -182,12 +182,16 @@
...
+					return;
 				if(key == "t")
-					window.location.href = "byond://winset?command=Say"
+					window.location.href = "byond://winset?command=Say";
 				else if(key == "o")
-					window.location.href = "byond://winset?command=OOC"
+					window.location.href = "byond://winset?command=OOC";
 				else if(key == "m")
-					window.location.href = "byond://winset?command=Me"
+					window.location.href = "byond://winset?command=Me";
```
Comment (LemonInTheDark) [inline html/statbrowser.html:194]: I uh

Comment (LemonInTheDark) [reply html/statbrowser.html:194]: oh js

Comment (Couls) [reply html/statbrowser.html:194]: semi colon 😳

File: code/modules/client/client_procs.dm [hunk 2]
```diff
@@ -998,8 +998,14 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	if(IsAdminAdvancedProcCall())
 		return
 	var/list/verblist = list()
+	var/list/verbstoprocess = verbs
+	if(mob)
+		verbstoprocess += mob.verbs
+		for(var/item in mob.contents)
+			var/atom/A = item
+			verbstoprocess += A.verbs
```
Comment (Rohesie) [inline code/modules/client/client_procs.dm:1006]:
```suggestion
		for(var/am in mob.contents)
			var/atom/movable/thing = am
			verbstoprocess += thing.verbs
```

File: code/modules/client/client_procs.dm [hunk 3]
```diff
@@ -998,8 +998,12 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
 	if(IsAdminAdvancedProcCall())
 		return
 	var/list/verblist = list()
+	var/list/verbstoprocess = verbs + mob?.verbs
```
Comment (Rohesie) [inline code/modules/client/client_procs.dm:1001]:
```suggestion
	var/list/verbstoprocess = verbs.Copy()
	if(mob)
		verbstoprocess += mob.verbs
```
This way you don't produce nulls in the list.

File: code/modules/client/client_procs.dm [hunk 4]
```diff
@@ -998,8 +998,12 @@ GLOBAL_LIST_INIT(blacklisted_builds, list(
...
+	for(var/AM in mob?.contents)
+		var/atom/movable/thing = AM
+		verbstoprocess += thing.verbs
```
Comment (Rohesie) [inline code/modules/client/client_procs.dm:1004]:
```suggestion

	var/list/verbstoprocess = verbs.Copy()
	if(mob)
		verbstoprocess += mob.verbs
		for(var/am in mob.contents)
			var/atom/movable/thing = am
			verbstoprocess += thing.verbs
```
This way you don't produce nulls in the list.

**Review by Rohesie — APPROVED** [review]

**Comment by ArcaneMusic** [comment]
Oh, wait, shit

**Pull request merged**